LayoutTests/media/media-source/media-managedmse-resume-after-stall.html

```diff
@@ -22,7 +22,8 @@
         loader = new MediaSourceLoader('content/test-fragmented-manifest.json');
         await loaderPromise(loader);
 
-        source = new ManagedMediaSource();
+        //source = new ManagedMediaSource();
+        source = new MediaSource();
```

Comment:
Is this a testing leftover? I guess we should create a ManagedMediaSource in a ManagedMediaSource test.

Reply:
Yes, sorry, I used WebKitGTK for some quick checks (there's no ManagedMediaSource there) and it's a leftover.